test/test_helpers.bash

```diff
@@ -46,7 +46,7 @@ install_mock_broken_plugin() {
 install_mock_plugin_repo() {
   local plugin_name=$1
   local location="${BASE_DIR}/repo-${plugin_name}"
-  cp -r "$BATS_TEST_DIRNAME/fixtures/dummy_plugin" "${location}"
```

Comment:
The `dummy` plugin was hardcoded into this helper function. It needed to be changed to test a new plugin with different name and subsequently broke the `plugin update` tests for updating to the remote default branch.

Reply:
nvm, been staring at screens too long